status-network-contracts/src/StakeManager.sol

```diff
@@ -76,6 +76,19 @@ contract StakeManager is
     mapping(address vault => address owner) public vaultOwners;
     /// @notice Flag to enable emergency mode.
     bool public emergencyModeEnabled;
+    address public guardian;
+
+    /// @notice Guardian role keccak256("GUARDIAN_ROLE")
+    bytes32 public constant GUARDIAN_ROLE = 0x55435dd261a4b9b3364963f7738a7a662ad9c84396d64be3365284bb7f0a5041;
+    /// @notice Rewards supplier role keccak256("REWARDS_SUPPLIER_ROLE")
+    bytes32 public constant REWARDS_SUPPLIER_ROLE = 0x1e153db16d10d4d642e290a54b15e41102f4abaeaa9448b47be3321a9c05a16c;
```

Comment:
Actually, I'm not sure it makes sense to introduce a role for this.
This would essentially allow for configuring multiple reward suppliers, which breaks the implicit invariant of the system that all Karma is distributed through the Karma contract (of which there's only ever one instance).

Reply:
Ah got it, I'll revert then

Reply:
I understood it to be a EOA role, so it's in fact the Karma contract?

Reply:
That is correct. Karma contract supplies the stakemanager with rewards